### PR TITLE
[P4-960] Person search logic

### PR DESCRIPTION
**Pull request by teneightfive**
Create logic to determine which person search step a user should see based on the location they are sending someone from.

## Sending someone from Police Custody Suite, Secure Training Centre or Secure Children's Home
police-journey

## Sending someone from Prison
prison-journey

### CODE REVIEW

File: app/move/controllers/create/base.js [hunk 1]
```diff
@@ -65,6 +65,21 @@ class CreateBaseController extends FormWizardController {
...
     next()
   }
+
+  saveValues(req, res, next) {
+    const { currentLocation } = req.session
+    const {
+      id: currentLocationId,
+      location_type: locationType,
+      can_upload_documents: canUploadDocuments,
+    } = currentLocation
```
Comment (barbaracassani) [inline app/move/controllers/create/base.js]: You can probably do the destructuring in one go -- is there a reason why there's an interim variable? (not that is very important at all!).

File: app/move/views/create/no-results.njk [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% extends "form-wizard.njk" %}
+
+{% block contentMain %}
+Can't create new prisoners
```
Comment (barbaracassani) [inline app/move/views/create/no-results.njk]: Just to be pedantic, we can put this in the locale files.

`{{ t("person::no_results_text") }}`

Comment (teneightfive) [reply app/move/views/create/no-results.njk]: Doh! These weren't meant to be committed! These are still WIP!

File: app/move/controllers/create/base.js (hunk 1 above)
Comment (teneightfive) [reply app/move/controllers/create/base.js]: Not sure what you mean? So just use `currentLocation.<property>` directly?

Comment (teneightfive) [reply app/move/controllers/create/base.js]: Oh I see! `req.session.currentLocation`! Yep, thanks

File: app/move/views/create/no-results.njk (hunk 2 above)
Comment (teneightfive) [reply app/move/views/create/no-results.njk]: Also, no pedantic at all. We should try to always use the translations for content.

**Pull request merged**